types/react-dual-listbox/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/react-dual-listbox",
-    "version": "2.2.9999",
+    "version": "4.0.9999",
```

Comment:
The PR description says 6.0, but here it says 4.0. Which is right?

Reply:
It is my mistake. Apologies. It should indeed be 6.0.9999. Fixing it.

Reply:
Please ignore my last push, I accidentally made a merge commit while trying to sync my repo. First time doing a PR for a big project like this... Apologies.

Reply:
@sandersn I have made the change. Please verify. Thank you very much for your time.